src/components/ModelDialog/ModalDialog.tsx

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 -2023 Vereniging van Nederlandse Gemeenten, Gemeente Amsterdam
+import type { ReactNode } from 'react'
+
+import { ModalContent } from './components/ModalContent'
+import { ModalHeader } from './components/ModalHeader'
+import { StyledModal } from './styled'
+
+export type ModalDialogProps = {
+  onClose: () => void
+  title: string
+  onConfirm?: () => void
+  children?: ReactNode
+  isConfirmation?: boolean
+  description?: string
+}
+
+const ModalDialog = ({
+  onClose,
+  onConfirm,
+  title,
+  isConfirmation,
+  description,
+  children,
+}: ModalDialogProps) => (
+  <StyledModal open onClose={onClose}>
+    <ModalHeader title={title} onClose={onClose} />
+    <ModalContent
+      description={description}
+      isConfirmation={isConfirmation}
+      onConfirm={onConfirm}
+      onClose={onClose}
+    >
+      {children}
+    </ModalContent>
```

Comment:
I would expect the footer here. Then you have a header, content and footer in one component

Reply:
Moved it.

Reply:
Done